use_cases/common/single_type/metrics/two_series_arithmetic.yaml

```diff
@@ -0,0 +1,71 @@
+title: Two Series Arithmetic
+
+description: >
+  A common operation operation involves combining two or more time series into a single
+  aggregate. Used most often in systems that require a single explicit metric name 
+  during query time with optional filters on tags. 
+
+inputs:
+  - metric series
+  -
+uses:
+  - type: Alerting
+    description: >
+      Used for simple, single threshold alerting on a combination of series. 
+
+  - type: Visualization
+    description: >
+      Used to reduce the amount of data displayed in a visualization and combine
+      complementary data.
+
+examples:
+  - language: PromQL
+    query: >
+      sum(http_requests_errors{handler="/api/comments"}) + 
+      sum(http_requests_success{handler="/api/comments"})
+    description: >
+      Selects two time series, `http_requests_errors` and `http_requests_success`,
+      filtering both where the tag `handler` has the value `/api/comments`, then
+      adding the results
+    references:
+      - https://prometheus.io/docs/prometheus/latest/querying/examples/#simple-time-series-selection
+      - https://prometheus.io/docs/prometheus/latest/querying/examples/#using-functions-operators-etc
+
+  - language: ASL
+    query: >
+      name,http_requests_errors,:eq,handler,/api/comments,:eq,:and,
+      name,http_requests_success,:eq,handler,/api/comments,:eq,:and,
+      :add
```

Comment:
It might also be worth calling out the behavior of some of these operations. For example in Atlas we treat `NaN` as a `0` for the purposes of `:add` and have `:fadd` for strict floating point addition. 

Also you could have avoided the duplication of the handler predicate similar to the UQL example:

```
name,http_requests_errors,:eq,:sum,
name,http_requests_success,:eq,:sum,
:add,
handler,/api/comments,:eq,:cq
```

Reply:
For `NaN` handling, etc. I'm thinking of having that in a callout at the language level. Or do you think it'd be best at each example?

Re the duplication, I'm thinking of trying to keep expressions as explicit as possible for one-to-one comparisons and maybe a callout for more concise examples. Would that work?